src/@utils/index.ts

```diff
@@ -17,3 +17,27 @@ export function sortAssets(items: Asset[], sorted: string[]) {
   })
   return items
 }
+
+export const isObjectLike = (object: any) => {
```

Comment:
suggestion(non-blocking): The function name `isObjectLike` could be more descriptive. A name like `isPlainObject` might be more appropriate, as it better describes the function's purpose. This naming is known from libraries like jQuery.